File: ejercicio2.java
// Definición de un tipo de dato abstracto
class persona {
    string nombre;
    string apellido;

    persona(string nombre, string appellido) {
        this.nombre = nombre;
        this.apellido = apellido;
    }
}

// Declarar un arreglo
public class main {
    public static void main(string[] args) {
        persona[] personas = new persona[3];
        personas[0] = new persona("Ana", "López");
        personas[1] = new persona("Luis", "García");
        personas[2] = new persona("Paola", "Valdez");
    }
}

File: ejercicio2.cpp
#include <iostream>
#include <string>
using namespace std;

// Tipo de dato abstracto
struct persona {
    string nombre;
    string apellido;
};

// Declarar arreglo
int main() {
    persona personas[3] = {
        {"Ana", "López"},
        {"Luis", "García"},
        {"Paola", "Valdez"}
    };

    return 0;
}